modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc

```diff
@@ -198,27 +218,31 @@ See link:https://quarkus.io/guides/building-native-image#using-a-multi-stage-doc
 Create a `meta.yaml` definition representing a {prod-short} plug-in that runs your custom back-end in a workspace Pod. For more information about `meta.yaml`, see xref:end-user-guide:what-is-a-che-theia-plug-in.adoc[].
 
 .meta.yaml
+====
 [source,yaml]
 ----
 include::example$telemetry/sample_meta.yaml[]
 ----
+====
 
-In most cases, a user would deploy this file to a corporate web server. For this guide, we create an Apache web server on OpenShift and host the plug-in there.
+In most cases, a user would deploy this file to a corporate web server. For this guide, create an Apache web server on OpenShift and host the plug-in there.
```

Comment:
Removing the `we` in this sentence changes its meaning. I'd put it back in.

Reply:
`we` is one of words we reject.

Reply:
You got a point: the complete procedure should be reviewed for modularity and language.
I added all errors reported by vale in the description of the PR to make it clear

Reply:
Maybe reword the whole sentence? 
"This guide demonstrates how to create an Apache..."
or
"This section guides you..."
?

Reply:
"This guide demonstrates how to create an Apache..." let's go.

Followup: https://github.com/eclipse/che/issues/19567